### PR TITLE
Add service lanes and show the new kuberpult homepage

### PR DESCRIPTION
**Pull request by tameremad**
image

* subscribing to the overview service at `index.tsx` is enough to track all changes and stream updates
* the first release cards belong to undeploy versions created locally.

### CODE REVIEW

**Review by hannesg — APPROVED** [review]

File: services/frontend-service/src/ui/App/index.tsx [hunk 1]
```diff
@@ -18,13 +18,40 @@ import { NavigationBar } from '../components/NavigationBar/NavigationBar';
...
-        <NavigationBar />
-        <div className="mdc-drawer-app-content">
-            <TopAppBar />
-            <PageRoutes />
+export const App: React.FC = () => {
+    const api = useApi;
+    React.useEffect(() => {
+        const subscription = api
+            .overviewService()
+            .StreamOverview({}) // TODO TE: add auth header
```
Comment (sven-urbanski-freiheit-com) [inline services/frontend-service/src/ui/App/index.tsx:30]: please add the ID of the story, where this `TODO` will be done @tameremad

Comment (sven-urbanski-freiheit-com) [reply services/frontend-service/src/ui/App/index.tsx:30]: (as a comment in the code)

Comment (tameremad) [reply services/frontend-service/src/ui/App/index.tsx:30]: I actually looked into this. I saw that we have a little complicated contexts setup for authentication. as I didn't want to move anything that could break I didn't move them. I will meet with Nisham to discuss them. I think some could be simplified.

Comment (tameremad) [reply services/frontend-service/src/ui/App/index.tsx:30]: if that proved too difficult I will refine a story for it

**Pull request merged**